plugins/snap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: snap
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/honk-ci/kubectl-snap
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/honk-ci/kubectl-snap/releases/download/0.1.0/kubectl-snap-0.1.0.tar.gz
+    sha256: "8df10f76e82f71b34c737d128b7b369e0daf641118577822872537e787a6755d"
+    bin: kubectl-snap
+  shortDescription: Randomly delete half of the pods in a namespace or cluster for chaos testing
+  caveats: |
+    To reiterate: This will delete half of the pods within the designated space. Currently, if there are pods that have failed or completed, they may get deleted and you may lose debug/log data. 
```

Comment:
Can you wrap lines in `caveats` and `description` at 80 chars, so it becomes readable on smaller terminals?

Reply:
Absolutely!